<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>631</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="baseSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolbarNewButton"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newDocument"/>
    <addaction name="openDocument"/>
    <addaction name="saveDocument"/>
    <addaction name="saveAsDocument"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="toolbarNewButton">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>Toolbar New Button</string>
   </property>
   <property name="toolTip">
    <string>asdasdasd</string>
   </property>
  </action>
  <action name="newDocument">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="openDocument">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="saveDocument">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsDocument">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/print.png</normaloff>:/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/bold.png</normaloff>:/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/italic.png</normaloff>:/icons/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/underline.png</normaloff>:/icons/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="notepad.qrc"/>
  <include location="notepad.qrc"/>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
